azure/terraform/modules/app_service/app_service.tf

```diff
@@ -60,6 +60,7 @@ resource "azurerm_app_service" "app_as" {
     "NOTIFY_API_KEY"                                 = data.azurerm_key_vault_secret.NotifyApiKey.value
     "ORDNANCE_SURVEY_CLIENT_PARAMS"                  = data.azurerm_key_vault_secret.ordnancesurveyclientparms.value
     "ORDNANCE_SURVEY_API_BASE_URL"                   = data.azurerm_key_vault_secret.ordnancesurveyapibaseurl.value
+    "PR_NUMBER"                                      = var.PR_NUMBER
```

Comment:
The app environment variable is uppercase but the terraform variable should be lowercase, as in `var.pr_number`

Reply:
I used capitals according to this doc  : 

"Then, in your Terraform configuration, you have to define the variables with capital letters.

Why? Because Azure DevOps will always transform pipeline variables to uppercase environment variables."